mono/metadata/Makefile.am

```diff
@@ -70,10 +70,10 @@ endif
 
 if SUPPORT_BDWGC
 if DISABLE_EXECUTABLES
-shared_bdwgc_libraries = libmonoruntimebdwgc.la
+shared_bdwgc_libraries = libmonoruntimebdwgc.la libmonoruntime-il2cpp-bdwgc.la
```

Comment:
I think Josh made libmonoruntime-il2cpp-bdwgc a separate built target so we don't need to call make an additional time